docs/fips.md

```diff
@@ -0,0 +1,38 @@
+# FIPS compliance
+
+The Federal Information Processing Standards (FIPS) are a set of U.S. government standards that ensure the security of computer systems for non-military government agencies and contractors. These standards specify how to perform cryptographic operations, such as encryption, hashing, and digital signatures. FIPS mode is a mode of operation that enforces these standards and rejects any non-compliant algorithms or parameters.
+
+Percona Server for MySQL implements the same level of FIPS support as MySQL. Percona Server for MySQL can run in FIPS mode if a FIPS-enabled OpenSSL library and FIPS Object Module are available at runtime or if compiled using a FIPS-validated version of OpenSSL. You can also receive this functionality by [building Percona Server for MySQL from source code](compile-percona-server.md).
+
+Percona Server for MySQL uses the same variables and values as MySQL. Percona Server for MySQL enables control of FIPS mode on the server side and the client side:
+
+* The `ssl_fips_mode` system variable controls whether the server operates in FIPS mode. This variable is disabled by default.
```

Comment:
As this system variable became deprecated and read-only in MySQL8.0.34 it no longer "controls" whether the server operates in FIPS mode, just "shows".
To "control", the "--ssl-fops-mode" server command line option should be used

Reply:
updated